core/controllers/profile_test.py

```diff
@@ -282,3 +285,29 @@ def test_profile_data_is_independent_of_currently_logged_in_user(self):
         response = self.get_json(
             '/profilehandler/data/%s' % self.EDITOR_USERNAME)
         self.assertEqual(response['user_bio'], 'My new editor bio')
+
+
+class FirstContributionDateTests(test_utils.GenericTestBase):
+
+    USERNAME = 'abc123'
+    EMAIL = 'abc123@gmail.com'
+
+    def test_contribution_datetime(self):
+        """Test the contribution date shows up correctly as nonexist."""
+        self.signup(self.EMAIL, self.USERNAME)
+        self.login(self.EMAIL)
+        self.user_id = self.get_user_id_from_email(self.EMAIL)
+        response_dict = self.get_json(
+            '/profilehandler/data/%s' % self.USERNAME)
+        self.assertEqual(response_dict['first_contribution_datetime'], None)
+
+        current_datetime = datetime.datetime.utcnow()
+        user_services.update_first_contribution_datetime(self.user_id,
+            current_datetime)
+
+        """Test the contribution date correctly changes to set date time."""
+        self.login(self.EMAIL)
+        response_dict = self.get_json(
+            '/profilehandler/data/%s' % self.USERNAME)
+        self.assertEqual(response_dict['first_contribution_datetime'],
```

Comment:
This second line is now over 80 char:
self.assertEqual(
            response_dict['first_contribution_datetime'],utils.get_time_in_millisecs(current_datetime))

Do you recommend I break it or just leave it as is.